FusionAuth/fusionauth-nodejs-react-example
# Updated FusionAuth Demo

I have updated the FusionAuth Demo utilizing NodeJS and ReactJS.

Last item that needs to be done is updating of the Readme files which will be done while the code is reviewed and functionality is checked.

Tested with:
- v1.9.2 FusionAuth
- v10.15.2 NodeJS

Functionality:
- Language Support (includes English + Spanish)
- Redux (for storing the local user data and language data)
- View user's own profile (limited data)
- Edit user's own profile (limited data)
- Add, edit, view, and delete own ToDo items
- Self register
- Verify 2FA for login
- Enable / disable 2FA
- Custom email templates
- Change password (logged in + logged out)
- Forgot password flow
